apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -1502,20 +1516,17 @@ private void HandleAckResponseForExactlyOnceDelivery(Task writeTask, List<string
                 // All methods below are local methods applicable to exactly once delivery subscriptions only.
                 // They are not used in non-exactly once delivery subscriptions.
 
-                // This method schedules the retry of ackIds with temporary errors and
-                // handles the permanent errors. Permanent errors are propagated to the caller for acks.
+                // This local method for exactly once delivery only, schedules the retry of ackIds with temporary errors.
```

Comment:
There's a verb missing in "This local method for exactly once delivery only" here and below.

Reply:
Added "used" at the relevant places.